src/autogluon_dashboard/plotting/interactive_df.py

```diff
@@ -17,15 +17,19 @@ def __init__(
         self.dataset = dataset
 
     def get_interactive_df(self) -> hvplot.Interactive:
-        return hvplot.bind(self._preprocess, self.framework, self.dataset).interactive(width=self.table_width)
+        return hvplot.bind(self._preprocess, framework=self.framework, dataset=self.dataset).interactive(
+            width=self.table_width
+        )
 
-    def _preprocess(*args) -> pandas.DataFrame:
-        df = args[0].dataset_to_plot
-        if args[2]:
-            df = get_df_filter_by_dataset(df, args[2])
-        if "All Frameworks" in args[1]:
+    def _preprocess(self, **kwargs) -> pandas.DataFrame:
```

Comment:
instead do `def _preprocess(self, dataset, framework, **kwargs)` so you can get them directly and they are expected parameters